Core/PixelExperiment.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  PixelExperiment.swift
+//  DuckDuckGo
+//
+//  Copyright © 2024 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+public enum PixelExperiment: String, CaseIterable {
+
+    fileprivate static let logic = PixelExperimentLogic {
+        Pixel.fire(pixel: $0)
+    }
+
+    /// When `cohort` is accessed for the first time after the experiment is installed with `install()`,
+    ///  allocate and return a cohort.  Subsequently, return the same cohort.
+    public static var cohort: PixelExperiment? {
+        logic.cohort
+    }
+
+    static var isExperimentInstalled: Bool {
+        return logic.isInstalled
+    }
+
+    static var allocatedCohortDoesNotMatchCurrentCohorts: Bool {
+        guard let allocatedCohort = logic.allocatedCohort else { return false }
+        if PixelExperiment(rawValue: allocatedCohort) == nil {
+            return true
+        }
+        return false
+    }
+
+    /// Enables this experiment for new users when called from the new installation path.
+    public static func install() {
+        // Disable the experiment until all other experiments are finished
+        // logic.install()
```

Comment:
To avoid having a PR open for weeks and keeping it conflict free, I'd rather merge it without enabling new Settings (without running an experiment). 

Once everyting is ready, I'd just open a new PR with this line uncommented and that would trigger the experiment after the merge.

Are there any strong objections about this approach?